tutorials/navigation/navigation_using_navigationpathqueryobjects.rst

```diff
@@ -70,3 +99,197 @@ has a large quantity of simultaneous agents that regularly update their paths.
+
+If ``simplify_path`` is enabled a variant of the Ramer-Douglas-Peucker path simplification algorithm is applied to the path.
+This algorithm straightens paths by removing less relevant path points depending on the used ``simplify_epsilon``.
+
+Path simplification helps with all kinds of agent movement problems in "open fields" that are caused by having many unnecessary polygon edges.
+E.g. a terrain mesh when baked to a navigation mesh can cause an excessive polygon count due to all the small (but for pathfinding almost meaningless) height variations in the terrain.
+
+Path simplification also helps with "steering" agents because they only have more critical corner path points to aim for.
+
+.. Warning::
+
+    Path simplification is an additional final post-processing of the path. It adds extra performance cost the query so only enabled the simplification when actually needed.
+
+.. note::
+
+    Path simplification is exposed on the NavigationServer as a generic function. It can be used outside of navigation queries for all kinds of position holding arrays as well.
+
+Path meta data
+--------------
+
+.. tip::
+
+    Disabling unneeded path meta data options can improve performance and lower memory consumption.
+
+A path query can return additional meta data for every path point.
+
+- Flag ``PATH_METADATA_INCLUDE_TYPES`` collects an array with the primitive information about the point owners, e.g. if a point belongs to a region or link.
+- Flag ``PATH_METADATA_INCLUDE_RIDS`` collects an array with the :ref:`RID<class_RID>` of the point owners. Depending on point owner primitive, these RIDs can be used with the various NavigationServer functions related to regions or links.
+- Flag ``PATH_METADATA_INCLUDE_RIDS`` collects an array with the ``ObjectIDs`` of the point owners. These object ids can be used with :ref:`@GlobalScope.instance_from_id()<class_@GlobalScope_method_instance_from_id>` to receive the node behind that object instance, e.g. a NavigationRegion or NavigationLink node.
+
+By default all path meta data is collected as this meta data can be essential for more advanced navigation gameplay.
+
+- E.g. to know what path point maps to what object or node owner inside the SceneTree.
+- E.g. to know if a path point is the start or end of a navigation link that requires scripted takeover.
+
+For the most basic path uses meta data is not always needed.
+Path meta data collection can be selectively disabled to gain some performance and reduce memory consumption.
+
+Excluding or including regions
+------------------------------
+
+.. tip::
+
+    Region filters can greatly help with performance on large navigation maps that are region partitioned.
+
+Query parameters allow to limit the pathfinding to specific region navigation meshes.
+
+If a large navigation map is well partitioned into smaller regions this can greatly help with performance as the query can skip a large amount of polygons at one of the earliest checks in the path search.
```

Comment:
```suggestion
If a large navigation map is well partitioned into smaller regions this can greatly help with performance as the
query can skip a large number of polygons at one of the earliest checks in the path search.
```